internal/bminventory/inventory_test.go

```diff
@@ -9256,9 +9354,22 @@ var _ = Describe("V2DownloadInfraEnvFiles", func() {
 			os.Unsetenv("EC_PRIVATE_KEY_PEM")
 		})
 
+		It("IPXE without mac with boot control", func() {
+			content := getResponseData("ipxe-script", false, true)
+			chainRegex := regexp.MustCompile(`^chain *(.*file_name=ipxe-script&mac=[$]{net0/mac})`)
+			match := chainRegex.FindStringSubmatch(strings.Split(string(content), "\n")[1])
+			Expect(match).NotTo(BeNil())
+
+			chainUrl, err := url.Parse(match[1])
+			Expect(err).NotTo(HaveOccurred())
+			tok := chainUrl.Query().Get("api_key")
+			_, err = bm.authHandler.AuthURLAuth(tok)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
 		It("signs the initrd ipxe-script url correctly", func() {
 			mockVersions.EXPECT().GetOsImage(common.TestDefaultConfig.OpenShiftVersion, gomock.Any()).Return(common.TestDefaultConfig.OsImage, nil).Times(1)
-			content := getResponseData("ipxe-script")
+			content := getResponseData("ipxe-script", true, false)
```

Comment:
None of these existing tests should have a mac passed. We need to ensure they work the same as they did before.